unity-renderer/Assets/ABConverter/WearablesCollectionClient.cs

```diff
@@ -0,0 +1,347 @@
+                }
+
+                throw new ArgumentException("Invalid arguments! You must pass -wearablesCollectionUrnId for dump to work!");
+            }
+            catch (Exception e)
+            {
+                log.Error(e.Message);
+            }
+        }
+        
+        /// <summary>
+        /// Dump all bodyshape wearables normally, including their imported skeleton 
+        /// </summary>
+        public static void DumpAllBodyshapeWearables()
+        {   
+            EnsureEnvironment();
+            
+            List<WearableItem> avatarItemList = GetWearableItems(BuildWearableCollectionFetchingURL(WearablesFetchingHelper.BASE_WEARABLES_COLLECTION_ID))
+                .Where(x => x.data.category == WearableLiterals.Categories.BODY_SHAPE)
+                .ToList();
+
+            Queue<WearableItem> itemQueue = new Queue<WearableItem>(avatarItemList);
+            var settings = new ClientSettings();
+            settings.skipAlreadyBuiltBundles = false;
+            settings.deleteDownloadPathAfterFinished = false;
+            settings.clearDirectoriesOnStart = false;
+            var abConverterCoreController = new ABConverter.Core(ABConverter.Environment.CreateWithDefaultImplementations(), settings);
+
+            abConverterCoreController.InitializeDirectoryPaths(true, true);
+            DumpWearableQueue(abConverterCoreController, itemQueue, GLTFImporter_OnBodyWearableLoad);
+        }
+        
+        public static void DumpSingleWearablesCollection(string collectionId, ClientSettings settings = null)
+        {
+            EnsureEnvironment();
+            
+            EnsureWearableCollections();
+
+            log.Info("Starting wearables dumping for collection: " + collectionId.Length);
+
+            if (settings == null)
+            {
+                settings = new ClientSettings();
+                settings.skipAlreadyBuiltBundles = false;
+                settings.deleteDownloadPathAfterFinished = false;
+                settings.clearDirectoriesOnStart = false;
+            }
+            
+            var abConverterCoreController = new ABConverter.Core(ABConverter.Environment.CreateWithDefaultImplementations(), settings);
+            
+            DumpWearablesCollection(abConverterCoreController, collectionId, (x) =>
+            {
+                log.Info($"Finished single wearables collection dumping");
+            });
+        }
+
+        /// <summary>
+        /// Dump all non-bodyshape wearables, optimized to remove the skeleton for the wearables ABs since that is
+        /// only needed for the body shapes (and the WearablesController sets it up for non-bodyshapes in runtime).
+        /// Each collection is dumped and converted sequentially as the amount of collections has grown more that we
+        /// can handle in 1 massive dump-conversion.
+        /// </summary>
+        public static void DumpAllNonBodyshapeWearables(Func<string, bool> collectionFilter = null)
```

Comment:
If we aren't using the `collectionFilter` argument maybe it's a good idea to just remove it?

Reply:
You are right, since now we have the other `DumpSingleWearablesCollection()` for that. I'll remove the filter!